docs/release-notes/8.13.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[release-notes-header-8.13.0]]
+== 8.13
+
+[discrete]
+[[release-notes-8.13.0]]
+=== 8.13.0
+
+[discrete]
+[[features-8.13.0]]
+==== Features
+
+* Allows you to define an entity's (such as a host's or user's) `Asset criticality`, which can affect risk scores ({pull}176815[#176815], {pull}176294[#176294], {pull}172417[#172417], {pull}176056[#176056]).
+* Information on the Data Quality dashboard now persists in {elastic-sec} rather than disappearing after each session ({pull}175673[#175673], {pull}173185[#173185]).
+* Adds line-by-line diffs to the rules upgrade flyout so you can see what's changed between versions ({pull}174564[#174564]).
+* Adds alert suppression to the Indicator Match rule type ({pull}174241[#174241]).
+
+
+[discrete]
+[[enhancements-8.13.0]]
+==== Enhancements
+
+* Adds a date and dataview selector to Analyzer regardless of where it appears in {elastic-sec} ({pull}176364[#176364]).
+* Enables the newly expanded Host and User details flyouts by default ({pull}175899[#175899]).
+* Updates the Alert details flyout's header UI ({pull}175075[#175075]).
+* Improves the UI for information panels appearing in Analyzer ({pull}172397[#172397]).
```

Comment:
@christineweng let me know if the following summary works, or if you'd prefer to focus on a particular change in the PR (e.g., the inline actions and search bar).

```suggestion
* Adds inline actions and a search bar to the left panel in the event analyzer UI and improves formatting issues ({pull}172397[#172397]).
```